app/lib/activitypub/activity/create.rb

```diff
@@ -283,6 +283,7 @@ def process_attachments
         RedownloadMediaWorker.perform_in(rand(30..600).seconds, media_attachment.id)
       rescue Seahorse::Client::NetworkingError => e
         Rails.logger.warn "Error storing media attachment: #{e}"
+        RedownloadMediaWorker.perform_in(rand(30..600).seconds, media_attachment.id)
```

Comment:
I would retry more aggressively than this, similar to the Sidekiq retry times? Network errors can be transient, and waiting for 5 minutes (at most) before the next retry the media seems to be a lot?

Reply:
Changed to enqueue immediately. The idea for the randomized wait is to not overwhelm the remote server, but this is less relevant when the cause for the issue is on our end.